pages/data-design/start-using-inspector.mdx

```diff
@@ -72,7 +78,7 @@ Find the tag snippet and instructions to send data <Link passHref href="/impleme
 
 ### Install Inspector SDK
 
-If you’re not using one of the supported CDPs you can send events data with an SDK. It will require one time setup and a new app release and takes 2-30 minutes, depending on the developer’s experience. We recommend asking a developer that is familiar with the existing event tracking setup to help with this setup.
+If you’re not using one of the supported CDPs you can send events data with an SDK. It will require one time setup and a new app release and takes 10-30 minutes, depending on the developer’s experience. We recommend asking a developer that is familiar with the existing event tracking setup to help with this setup.
```

Comment:
This paragraph indicates that you should use the SDK if available, rather than a CDP.

I'm not sure that's the messaging we want to go for here. I'd suggest flipping it around. If you are using one of our CDPs you should go for that integration, because it's quicker and doesn't require developer buy-in!

Reply:
Wait, the first sentence says: 

> If you’re not using one of the supported CDPs you can send events data with an SDK.

That reads to me that we are indeed pitching the CDPs as the primary path and the SDK as a fallback. Do you not agree @logason?